sig-architecture/api-review-process.md

```diff
@@ -85,94 +61,83 @@ Because expert reviewer bandwidth is extremely limited, the process provides a c
-
-Timing of API review requests matters.  The larger the change the more time that must be afforded.  New API resources (aka Kinds) may require significantly more thought than single field additions.  API reviews that are requested too late in a release cycle may not complete in time to make the release.  Plan ahead. Also, if you are changing an approved API, you must consult with the [kubernetes-api-reviewers@googlegroups.com](mailto:kubernetes-api-reviewers@googlegroups.com) list to ensure it is still consistent with the approvals already granted. From a process perspective, you would request a new review in that case.
-
-New APIs (groups or Kinds) or substantial changes require KEPs.  Major changes without KEPs will be rejected.
-
-1. Create an API Review request issue in [https://github.com/kubernetes-sigs/architecture-tracking](https://github.com/kubernetes-sigs/architecture-tracking) - the work will be tracked in the corresponding project board
-
-    1. Provide the following information:
-
-        1. Submitter GitHub IDs
-
-        2. Links to code/issues/documentation/KEP (keep in mind that all code will need review as well, as translation errors between proposals/markdown can and do happen)
-
-        3. One line description of the purpose
-
-        4. Acknowledgement of reading/following the API conventions document
-
-2. If the work is currently in GitHub kubernetes/kubernetes (or other explicitly-critical repository) as either a pull request or issue, add a link to the PR/Issue pointing at the API Review request issue
-
-3. An approver or moderator will adjust the prioritization of the request in the backlog, assign an approver (or potentially aspiring reviewer), and add the label status/assigned-to-reviewer (see *training reviews* below for the aspiring reviewer workflow)
-
-4. Assigned approver will either start the review, work with the moderator to schedule face-to-face discussions during SIG-Architecture, or lower its priority in the backlog. They may also request that other reviewers be involved at their discretion.
-
-5. Once the review is completed, the report will be made available to the submitters and one of the three labels applied: APPROVED  REVIEWED or UNAPPROVED These may also be added after face-to-face discussions.
-
-6. Either an appeal happens, or the issue is closed
+# Mechanics
+
+0. Requesters should complete the pre-review checklist:
+    * The goal of the proposed change has agreement from the appropriate sub-project owners and/or SIG leads
+    * A [KEP](https://github.com/kubernetes/enhancements/blob/master/keps/0001-kubernetes-enhancement-proposal-process.md) and tracking issue in [kubernetes/enhancements](https://github.com/kubernetes/enhancements/) has been created for changes within the kubernetes-* org introducing:
+        * Any new resource type
+        * Any new version of a stable API
+        * Any new functionality added to a stable API as defined by SIG Architecture and the API Reviewers
```

Comment:
Should also include `Any change to the meaning, validation, or behavior of a field`